altianalysis/run_difference_with_gpao.py

```diff
@@ -67,7 +66,13 @@ def create_main_gpao_project(
     project_name: str,
 ) -> Project:
 
-    logging.debug(f"Create GPAO projects to compute difference maps with rge alti: {primary_dir}.")
+    if secondary_dir:
+        logging.debug(
+            f"Create GPAO projects to compute difference maps file by file between : \
```

Comment:
un micro-pinaillage pour l'absence d'espace avant `:` en anglais 

```suggestion
            f"Create GPAO projects to compute difference maps file by file between: \
```